client/app/config/routes.js

```diff
@@ -0,0 +1,18 @@
+import HomeView from '../views/HomeView';
+import TestView from '../views/TestView';
+import TabBarView from '../views/TabBarView';
+import TemplateView from '../views/TemplateView';
+
+const TabBarRoutes = {
+    FriendsView: TemplateView,
+    DiningHallsView: TestView,
+    FavoritesView: TemplateView
+}
+
+const Routes = {
```

Comment:
Ditto above

Reply:
Yep, we will add to this for routes not directly associated with the `TabNavigator`. This needs to be discussed though, because pushing routes not in the `TabNavigator` could remove the tab bar when we move screens.